eng/Versions.props

```diff
@@ -32,7 +32,7 @@
     <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>17.8.9-preview</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
     <MicrosoftVisualStudioShellPackagesVersion>17.8.36711</MicrosoftVisualStudioShellPackagesVersion>
     <!-- The version of MSBuild that we reference for things depending on MSBuildWorkspace -->
-    <RefOnlyMicrosoftBuildPackagesVersion>16.10.0</RefOnlyMicrosoftBuildPackagesVersion>
+    <RefOnlyMicrosoftBuildPackagesVersion>17.3.2</RefOnlyMicrosoftBuildPackagesVersion>
```

Comment:
We may want to just get rid of this in favor of using the other variable below, but might be good to do this one step at a time.

Reply:
Yes, I think we should align onto a single version. Roslyn currently uses 17.5.0 which isn't in [SBRP](https://github.com/dotnet/source-build-reference-packages), so we would need to add it there.